recipe/meta.yaml

```diff
@@ -5,44 +5,60 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/nco/nco/archive/{{ version }}.tar.gz
-  sha256: 5ba930f00a0e9775f85748d145acecfe142f917c180de538b2f8994788446cf8
+  # url: https://github.com/nco/nco/archive/{{ version }}.tar.gz
+  # sha256: 5ba930f00a0e9775f85748d145acecfe142f917c180de538b2f8994788446cf8
+  # Just testing before we have a new release.
+  git_url: https://github.com/nco/nco.git
+  git_branch: master
+  patches:
+    # See https://github.com/conda-forge/nco-feedstock/pull/48#issuecomment-351592929
+    - cmake.patch
+    - strcasecmp.patch
```

Comment:
These two patches are in git master now

Reply:
I updated to reflect that. Thanks!

@czender if this passes it would be nice to have a new release so we can make this "official."

Reply:
I merged the patches upstream. @ocefpaf does everything work with Windows? Let me know. It would be nice, though not a deal-breaker, to have GSL functionality in the Windows version. Once this is done, or progress stalls, I'll happily tag and release NCO 4.7.1, which has no other major features pending. That should be the official beginning of availability on Windows conda-forge.

Reply:
> I merged the patches upstream.

Saw that. Thanks!

> @ocefpaf does everything work with Windows?

We are testing the commands

```
ncks -H --trd -v one in.nc
ncks --help
ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"
```

and those are working fine. I'll add a simple `ncap2` test, like for Linux. Do you suggest anything else?

Reply:
BTW, we can set Appveyor up in the `nco` repo that would build this recipe and run the test. That would help catch bugs on Windows early.

Reply:
Please do set up Appveyor, if you have time.
I don't particularly expect the Bash shell scripts ncclimo and ncremap to work with Windows, and I have no way to test that, so let's not worry about getting them working until after 4.7.1 is released.
I'm unclear on the status of GSL in the Windows build. Seems like there was a link problem. This would be a good test for any platform on which GSL is thought to work:
`zender@aerosol:~$ ncap2 -O -v -s 'erf_one=float(gsl_sf_erf(1.0f));print(erf_one,"%g")' ~/nco/data/in.nc ~/foo.nc

0.842701`